addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1259,13 +1262,15 @@ const DatetimePickerUserValueWidget = InputUserValueWidget.extend({
             calendarWeeks: true,
             defaultDate: moment().format(),
             icons: {
+                clear: 'far fa-delete',
```

Comment:
`far` ? You handled the migration to font-awesome 5 in this task as well ? ^^

Reply:
Haha I for sure don't handle the revert... The button disappear if I use fa Did not digged further as you told me there was an ongoing task for it could you link it to me so I either handle it or make sure we merge it before this task